### PR TITLE
Chard's Card

### PR DESCRIPTION
**Pull request by ChardPeter**
Added a new card and modified the content with my info

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+        <div class="card">
+          <p class="name">Chard Peter</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://x.com/chardpeter1?s=21" target="_blank">Your handle</a>
```
Comment (TenTraicion) [inline index.html:128]:
```suggestion
            <a href="https://x.com/chardpeter1" target="_blank">@chardpeter1</a>
```

File: index.html [hunk 2]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com/chardpeter" target="_blank">Your handle</a>
```
Comment (TenTraicion) [inline index.html:130]:
```suggestion
            <a href="https://www.github.com/chardpeter" target="_blank">@chardpeter</a>
```

File: index.html [hunk 3]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">I am an amazing honest man. Sweet and simpile. An introverty extrovert :D</p>
+          <div class="resources">
+            <p>FreeCodeCamp, Udemy and In person conferences</p>
```
Comment (TenTraicion) [inline index.html:134]:
```suggestion
            <p>3 Useful Dev Resources</p>
```

File: index.html [hunk 4]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+            <ul>
+              <li>
+                <a href="Freecodecamp.org" target="_blank" title="First Resource">Freecodecamp</a>
```
Comment (TenTraicion) [inline index.html:137]:
```suggestion
                <a href="https://freecodecamp.org" target="_blank" title="First Resource">Freecodecamp</a>
```

File: index.html [hunk 5]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="udemy.com" target="_blank" title="Second Resource">udemy</a>
```
Comment (TenTraicion) [inline index.html:140]:
```suggestion
                <a href="https://udemy.com" target="_blank" title="Second Resource">Udemy</a>
```

File: index.html [hunk 6]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="Google.com" target="_blank" title="Third resource">Google</a>
```
Comment (TenTraicion) [inline index.html:143]:
```suggestion
                <a href="https://google.com" target="_blank" title="Third resource">Google</a>
```

File: index.html [hunk 7]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+        <div class="card">
```
Comment (TenTraicion) [inline index.html:124]:
```suggestion

         <!-- ________ *Peter* Contributor card START________ -->
        <div class="card">
```

File: index.html [hunk 8]
```diff
@@ -121,7 +121,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="udemy.com" target="_blank" title="Second Resource">udemy</a>
+              </li>
+              <li>
+                <a href="Google.com" target="_blank" title="Third resource">Google</a>
+              </li>
+            </ul>
+          </div>
+        </div>
```
Comment (TenTraicion) [inline index.html:147]:
```suggestion
        </div>
        <!-- ________ *Peter* Contributor card END________ -->
        
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Hello there @ChardPeter, Thanks for your contribution. Kindly make the requested changes and commit. Thanks.

**Review by kamranzafar4343 — APPROVED** [review]
looks fine to me

**Comment by TenTraicion** [comment]
> looks fine to me

@kamranzafar4343, Don't mislead people. There are project maintainers who recheck every single commit before approving and merging them.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@ChardPeter, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**